frontend/src/components/Footer/footerStyle.js

```diff
@@ -0,0 +1,29 @@
+import { Link } from 'react-router-dom'
+import { styled } from 'goober'
+
+export const Container = styled('footer')`
+  display: flex;
+  align-items: center;
+  gap: 1em;
+  padding-inline: 1em;
+  position: fixed;
```

Comment:
Sorry but I don't like how the footer is always onscreen, it should just be at the bottom of the page